envelope.js

```diff
@@ -35,7 +35,8 @@ module.exports = function Envelope (keystore, state) {
       }
       if (isFeed(recp)) {
         if (recp === state.keys.id) return keystore.self.get() // use a special key for your own feedId
-        else return keystore.encryptionKeys(state.keys.id, [recp])[0]
+        //else return keystore.encryptionKeys(state.keys.id, [recp])[0]
+        else return keystore.dm.get(state.keys.id, recp)
```

Comment:
not sure yet if this change helped/works. we're just getting the below error and then a freeze. it's probably/maybe related to unboxing, so it's probably better to try to get the unbox test working first

```
% node test/api/add-new-author-listener.test.js
TAP version 13
# addNewAuthorListener
not ok 1 TypeError: Cannot read properties of undefined (reading 'key')
  ---
    operator: fail
    at: Test.<anonymous> (/home/me/prj/ssb/ssb-tribes/test/api/add-new-author-listener.test.js:61:7)
    stack: |-
      Error: TypeError: Cannot read properties of undefined (reading 'key')
          at Test.assert [as _assert] (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:273:48)
          at Test.bound [as _assert] (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:85:17)
          at Test.fail (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:366:7)
          at Test.bound [as fail] (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:85:17)
          at Test.<anonymous> (/home/me/prj/ssb/ssb-tribes/test/api/add-new-author-listener.test.js:61:7)
  ...
ok 2 admin = returns expected newAuthors
ok 3 admin = returns expected groupId
```

Reply:
hopefully fixed here https://github.com/ssbc/ssb-tribes/pull/82#discussion_r1326365164